apps/openchallenges/edam-etl/Dockerfile

```diff
@@ -11,10 +11,11 @@ RUN pip install poetry \
   && poetry install --with prod --no-root --no-interaction --no-ansi \
   && pip cache purge
 
+
 WORKDIR /
 COPY docker-entrypoint.sh ./
 RUN chmod +x docker-entrypoint.sh
 
 ENTRYPOINT ["/docker-entrypoint.sh"]
 
-CMD python "${APP_DIR}/src/main.py"
\ No newline at end of file
+CMD python "${APP_DIR}/src/main.py"
```

Comment:
@tschaffter I missed this the first time around, but should this be in exec form?  Or does it not matter?

From the [docs](https://docs.docker.com/reference/dockerfile/#cmd):
> If CMD is used to provide default arguments for the ENTRYPOINT instruction, both the CMD and ENTRYPOINT instructions should be specified in the [exec form](https://docs.docker.com/reference/dockerfile/#exec-form).

Reply:
This improvement is also suggested by Hadolint, which will be added to our development workflow soon. For now it should still be addressed.